.gitlab-ci.yml

```
@@ -160,16 +215,46 @@ sync:
     - develop
   except:
     - schedules
-  tags:
-    - private_ci
-    - cpu
 
+trigger_pipeline:
+  stage: trigger_pipeline
+  extends:
+    - .pr_condition
+    - .use_gko-nocuda-gnu9-llvm8
+  variables:
+    STATUS_CONTEXT: "quick"
+  script:
+    - PR_ID=$(curl "https://api.github.com/search/issues?q=sha:${CI_COMMIT_SHA}"
```

Doesn't it need to be a silent curl?
```suggestion
    - PR_ID=$(curl -s "https://api.github.com/search/issues?q=sha:${CI_COMMIT_SHA}"
```

---

it seems to work with jq, but I add silent mode to those curl with jq or the results used as variable to avoid some possible issues from progress bar